packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -45,11 +49,15 @@ const getDisplayMessage = (
 		return ODIE_EMAIL_FALLBACK_MESSAGE_CONTENT;
 	}
 
+	if ( isErrorMessage && ! isUserEligibleForPaidSupport ) {
+		return ODIE_ERROR_MESSAGE_NON_ELIGIBLE;
+	}
```

Comment:
I think it'd be better to handle the `isErrorMessage` scenario in just one `if` clause.

```suggestion
	if ( isErrorMessage ) {
		return isUserEligibleForPaidSupport ? ODIE_ERROR_MESSAGE : ODIE_ERROR_MESSAGE_NON_ELIGIBLE;
	}
```